### PR TITLE
Fix forwarder test

### PR DESCRIPTION
**Pull request by mk13**
Uncomment the failing test and add fixes to no longer fail.

### CODE REVIEW

File: lib/support/forwarder.dart [hunk 1]
```diff
@@ -89,11 +89,11 @@ class WebDriverForwarder {
       if (endpoint.startsWith('/')) {
         endpoint = endpoint.substring(1);
       }
-      Map<String, dynamic> params;
+      Map<dynamic, dynamic> params;
       if (request.method == 'POST') {
         String requestBody = await utf8.decodeStream(request);
         if (requestBody != null && requestBody.isNotEmpty) {
-          params = json.decode(requestBody) as Map<String, dynamic>;
+          params = json.decode(requestBody) as Map<dynamic, dynamic>;
```
Comment (kevmoo) [inline lib/support/forwarder.dart]: every map coming out of json.decode should be `Map<String, dynamic` – are you sure this is a problem?

Comment (mk13) [reply lib/support/forwarder.dart]: The previous tests shows errors in casting from Map<String, dynamic>, so I believe these casts to be the culprits. The errors were thrown when trying to cast Map<dynamic, dynamic> to Map<String, dynamic>. This also puts the code in sync with internal.

Comment (jingbian) [reply lib/support/forwarder.dart]: What we get is `_InternalLinkedHashMap<dynamic, dynamic>`. We can either convert it to `Map<String, dynamic>` with `retype()`, or just treat it as `Map<dynamic, dynamic>` here, which is enough.

One possible related thread (though for Flutter): https://groups.google.com/forum/#!msg/flutter-dev/H8dDhWg_c8I/_Ql78q_6AgAJ

Comment (kevmoo) [reply lib/support/forwarder.dart]: `_InternalLinkedHashMap<dynamic, dynamic>` from jsonDecode is super weird. Are we absolutely sure?

Honestly, it doesn't matter that much. I'd just drop the type arguments and go with `as Map;` then...

**Comment by mk13** [comment]
Submitting code, there's some timeout-based failure, but this is fine.

**Comment by jingbian** [comment]
Related to this rule: https://www.dartlang.org/guides/language/effective-dart/design#prefer-annotating-with-dynamic-instead-of-letting-inference-fail

Should we actually prefer using `dynamic` in this?

**Comment by kevmoo** [comment]
I don't think it's a big deal either way, honestly...

**Pull request merged**